shared/actions/chat2/index.tsx

```diff
@@ -2807,6 +2808,26 @@ const onChatMaybeMentionUpdate = (state, action: EngineGen.Chat1ChatUiChatMaybeM
   })
 }
 
+const onChatGetCoordinate = (state, action: EngineGen.Chat1ChatUiChatGetCoordinatePayload) => {
+  const response = action.payload.response
+  if (isMobile) {
+    navigator.geolocation.getCurrentPosition(
+      pos =>
+        response.result({accuracy: pos.coords.accuracy, lat: pos.coords.latitude, lon: pos.coords.longitude}),
+      err => logger.warn(err.message),
+      {enableHighAccuracy: true, maximumAge: 0, timeout: 30000}
+    )
+  } else {
+    // desktop doesn't really work, so just stick us in SF
+    response.result({
```

Comment:
its possible to make this work, are we going to skip supporting it?

Reply:
Might be possible, but I hit a bunch of snags so just decided against it for this PR.